Classes/Middleware/TypoScriptRenderingMiddleware.php

```diff
@@ -73,8 +73,12 @@ public function process(ServerRequestInterface $request, RequestHandlerInterface
      */
     private function amendContentType(ResponseInterface $response, string $requestedContentType): ResponseInterface
     {
-        $originalContentTypeHeader = $response->getHeader('Content-Type')[0];
-        if (strpos($originalContentTypeHeader, self::defaultContentType) === 0 && strpos($originalContentTypeHeader, $requestedContentType) === false) {
+        $originalContentTypeHeader = $response->getHeader('Content-Type')[0] ?? '';
+        if (
+            !$originalContentTypeHeader
```

Comment:
Please let's have a proper string check instead of treating `$originalContentTypeHeader` like a boolean.

Also, please let's have a pair of parenthesis so the reader does not need to remember that `&&` has a stronger precedence than `||`.